tests/polyskel_convex_test.py

```diff
@@ -0,0 +1,226 @@
+# coding=utf-8
+"""Classes for computing straight skeleton for 2D polygons."""
+from __future__ import division
+
+import pytest
+
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry2d.pointvector import Point2D
+from ladybug_geometry.geometry2d.line import LineSegment2D
+from ladybug_geometry import polyskel as lb_polyskel
+
+from pprint import pprint as pp
+import numpy as np
+
+# TODO Figure out if we need this later
+# # For comparison
+# import sys
+# paths = ['/app/polyskel/']
+# for path in paths:
+#     if path not in sys.path: sys.path.insert(0,path)
+# import polyskel as orig_polyskel
+
+#TODO: move to updated polyskel once we import lb-geom?
+def skeleton_edges(skeleton):
+    """
+    Consumes list of polyskeleton subtrees.
+    Skeleton edges are the segments defined by source point and each sink points.
+
+    Args:
+        skeleton: list of polyskel.Subtree, which are namedTuples of consisting of
+        a source point, and list of sink points.

```

Comment:
Just a note that the second line for an input argument description should always be indented.